### DISCUSSION TITLE
Force-running seeds, in addition to "normal tests"

### DISCUSSION THREAD

**Discussion by stdedos** [dubzzz/fast-check#4406] [Q&A] [answered]
I have found a couple of seeds that have triggered issues, and I have fixed those issues.

Can I now force `fast-check` to always-run those `{ seed: -1108094780, path: '2:0:1:0:0:0:0:0:0:3:1:4:0:0:2:0:2:0:0:1:0:0:1:0:1:0:1:0:0:2:1:0', endOnFailure: true, }` cases, **in addition to** "normal" `fast-check` testing?
Would that make sense?

**Comment by dubzzz** [comment, marked as answer]
In order to do so I'd rather use the [examples](https://fast-check.dev/api-reference/interfaces/Parameters.html#examples) option that can be passed to `assert`.

Compared to "seed+path" which has been designed to simplify the replay for troubleshoot, this one has been designed to persist test cases forever. It will not break on minor bumps of fast-check while "seed+path" might not led to the same case after a bump on the minor.

**Reply by stdedos** [reply]
Would you contribute some _examples_ to the documentation (_pun intended_)?

It didn't take much time to deduce that

```ts
            fc.asyncProperty(fc.date(), fc.date(), async (start, end) => {
```

requires

```ts
            {
              examples: [
                // Validates the `` logic.
                [
                  new Date('1970-01-01T00:00:00.000Z'),
                  new Date('-000001-12-31T23:59:59.999Z'),
                ],
                [
                  new Date('-000002-12-31T23:59:59.999Z'),
                  new Date('-000001-12-31T23:59:59.999Z'),
                ],
                [
                  new Date('1970-01-01T00:00:00.000Z'),
                  new Date('1970-01-01T00:00:00.001Z'),
                ],
              ],
            },
```

_but I'd be nicer if I could've "just known" that, instead of deducing it_
